server/@types/approved-premises/index.d.ts

```diff
@@ -1,11 +1,41 @@
 declare module 'approved-premises' {
   export type Premises = schemas['Premises']
 
-  export type TableCell = { text: string; attributes?: { [key: string]: string } } | { html: string }
+  export interface HtmlAttributes {
+    [key: string]: string
+  }
+
+  export interface TextItem {
+    text: string
+  }
+
+  export interface HtmlItem {
+    text: string
```

Comment:
I think this should be `html`

Reply:
wishful thinking but it would be ideal if we could download GDS DS types from an NPM package 😄

Reply:
I'm planning on extracting these at some point I think, would be really useful for other folks I'm sure